examples/cluster/plot_segmentation_toy.py

```diff
@@ -63,25 +68,36 @@
 img = img.astype(float)
 img += 1 + 0.2 * np.random.randn(*img.shape)
 
+# %%
 # Convert the image into a graph with the value of the gradient on the
 # edges.
 graph = image.img_to_graph(img, mask=mask)
 
-# Take a decreasing function of the gradient: we take it weakly
-# dependent from the gradient the segmentation is close to a voronoi
+# %%
+# Take a decreasing function of the gradient resulting in a segmentation
+# that is close to a Voronoi partition
 graph.data = np.exp(-graph.data / graph.data.std())
 
-# Force the solver to be arpack, since amg is numerically
-# unstable on this example
+# %%
+# Here we perform spectral clustering using the arpack solver since amg is
+# numerically unstable on this example. We then plot the results.
 labels = spectral_clustering(graph, n_clusters=4, eigen_solver="arpack")
 label_im = np.full(mask.shape, -1.0)
 label_im[mask] = labels
 
-plt.matshow(img)
-plt.matshow(label_im)
+fig, axs = plt.subplots(nrows=1, ncols=2, figsize=(10, 5))
+axs[0].matshow(img)
+axs[1].matshow(label_im)
+
+plt.show()
```

Comment:
Thanks for including this! I find it makes the output more presentable. Would you mind also replacing the second plotting section (lines 114-117) with this as well for consistency?

Reply:
Of course! Thanks for catching that, I had missed it.